scripts/setup-ubuntu.sh

```diff
@@ -52,31 +49,86 @@ sudo apt install -y \
   liblzo2-dev \
   protobuf-compiler
 
-function install_folly {
-  local NAME="folly"
+function run_and_time {
```

Comment:
maybe not on this PR, but it would be nice to move this common code to some sort of library at some point.